src/api/app/views/webui/feeds/notifications.rss.builder

```diff
@@ -0,0 +1,20 @@
+xml.rss version: '2.0' do
+  xml.channel do
+    xml.title "#{@user.realname} (#{@user.login}) notifications"
+    xml.description "Event's notifications from #{@configuration['title']}"
+    xml.link url_for only_path: false, controller: 'main', action: 'index'
+    xml.language "en"
+    xml.pubDate Time.now
+    xml.generator @configuration['title']
+
+    @notifications.each do |notification|
+      xml.item do
+        xml.title notification.title
+        xml.description notification.description
```

Comment:
You could move the rendering from the model to here and do something like:

```ruby
xml.description render_to_string("event_mailer/#{event.template_name}", layout: false, format: :txt, assigns: { host: ::Configuration.obs_url, configuration: ::Configuration.first }, locals: { event: event})
```

I'm not sure if its `render` or `render_to_string` that you need..